save-plugins/fix-plugin/README.md

```diff
@@ -46,13 +46,14 @@ suiteName = "DocsCheck"
 execFlags="-F"
 testFilePattern="*Test.kt"
 expectedFilePattern="*Expected.kt"
-batchSize = 1
+batchSize = 1 # (default value)
+separator = ", " # (default value)
 resourceNameTestSuffix = "Test" # (default value)
 resourceNameExpectedSuffix = "Expected" #(default value)
 ```
 
 When executed from project root (where `save.propertes` is located), SAVE will cd to `rootDir` and discover all pairs of files
 matching `testFilePattern` and `expectedFilePattern` with same prefix. It will then execute `$execCmd $testFile`. `batchSize` it controls how many files execCmd will process at a time. (since we specified
-`batchSize = 1`, it will provide inputs one by one) and compare its stdout (as per `output` option) with respecting `$expectedFile`.
+`batchSize = 1`, it will provide inputs one by one) and compare its stdout (as per `output` option) with respecting `$expectedFile`. `separator` is a setting for batchSize.
```

Comment:
Please add more description: which requirements of `execCmd` can be satisfied by changing `separator`?